.github/workflows/cmake.yml

```diff
@@ -40,18 +40,17 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: configure
-        run: cmake -S . -B build -G "Ninja" -DCMAKE_C_COMPILER_LAUNCHER=sccache -DCMAKE_CXX_COMPILER_LAUNCHER=sccache ${{ matrix.flags }}
+        run: cmake --preset with-java-sccache ${{ matrix.flags }}
```

Comment:
Mac doesn't build Java. While the preset is overriden, I'd prefer to avoid overriding options like this.